spec/big_brother/health_fetcher_spec.rb

```diff
@@ -60,6 +60,37 @@
 HTTP/1.0 200 OK
 Connection: close
 
+[{"aggregated_health":0,"count":1,"lb_ip_address":"load1.stq","lb_url":"http://load1.stq:80/lvs.json","health":0}]
+HTTP
+      BigBrother::HealthFetcher.interpol_status(
+        [
+          Factory.node(:address => "127.0.0.1", :port => 8083, :interpol => true, :path => '/fwmark'),
+          Factory.node(:address => "127.0.0.1", :port => 8081, :interpol => true, :path => '/fwmark'),
+          Factory.node(:address => "127.0.0.1", :port => 8082, :interpol => true, :path => '/fwmark'),
+        ],
+        'test'
+      ).should == [
+        {"aggregated_health" => 0,"count" => 1,"lb_ip_address" => "load1.stq","lb_url" => "http://load1.stq:80/lvs.json","health" => 0}
+      ]
+    end
+
+    it "works with multiple interpol nodes and returns data from one even if a node is down or slow" do
```

Comment:
The title of this spec doesn't reflect it's content

Reply:
Do you have a preference on the title? One of the three nodes used in the test is intentionally made to be slow, so I felt like it was fitting, but happy to adjust!

Reply:
Sorry my bad, completely forgot how `StubServer` works